### ISSUE TITLE
Lock Submission Branch, Find Host for Post Jam Builds

### ISSUE THREAD

**Issue by MaverickLoneshark** [qjones45000/JustLetMeSleep#13]
- [ ] Need to add branch protection to `gh-pages` branch (basically copy the same thing we did for `master` but for `gh-pages`, @qjones45000 )
- [ ] ~Need to find hosting for updated versions (think you can give this puppy a home, @Tranzorro?)~

**Comment by MaverickLoneshark** [comment]
Actually, we could setup the github site to host the submission AND Post Jam, so never mind (my bad).

For Submission Branch: root url or `/index.html` (nothing changes)
For Updated Version: `/latest.html` (or something)
 - [ ] rename `index.html` to `latest.html` (or something)
 - [ ] use different Build folder name by changing references in `latest.html` (or whatever it's called)
 - [ ] update readme to include URL to `/latest.html`

Basically the two build folders, `index.html`, and `latest.html` will all be in the same directory, but we will add a URL to the readme for the updated version as well. Piece of cake ;-)
